lightning/src/ln/channelmanager.rs

```diff
@@ -1551,10 +1553,28 @@ impl ChannelDetails {
 			inbound_htlc_minimum_msat: Some(context.get_holder_htlc_minimum_msat()),
 			inbound_htlc_maximum_msat: context.get_holder_htlc_maximum_msat(),
 			config: Some(context.config()),
+			channel_shutdown_state: Some(context.shutdown_state()),
 		}
 	}
 }
 
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+/// Further information on the details of the channel shutdown
```

Comment:
```suggestion
/// Further information on the details of the channel shutdown. 
```

And something about what happens to force-closed channels?

Reply:
added some descriptions and another test for force closure of channels